### ISSUE TITLE
Consider creating a "Usage" section in the README

### ISSUE THREAD

**Issue by Shnatsel** [Byron/google-apis-rs#346]
Once I have found the repository, I had some trouble figuring out what are the names of the generated crates actually are. I have browsed the source code, but could not easily find them. 

Only the next day I found this in the README:

> To find a library of your interest, you might want to proceed looking at the [API documentation index](http://byron.github.io/google-apis-rs).

Once I found that link, everything was pretty straightforward.

Perhaps it should be featured more prominently, under a "Usage" heading in the readme.md or some such?

**Comment by Byron** [comment]
I totally agree! It should be straightforward to adjust one of the templates to make such improvement and a contribution is very welcome.
